avocado/core/job.py

```diff
@@ -254,7 +255,8 @@ def _make_test_runner(self):
             test_runner_class = runner.TestRunner
 
         self.test_runner = test_runner_class(job=self,
-                                             result_proxy=self.result_proxy)
+                                             result_proxy=self.result_proxy,
+                                             result=self.result)
```

Comment:
Yes, the last patch uses the `job.result`, but it'd be nice if you could also include the patch to remove `result_proxy` within this patchset (after you update all the result plugins). I'd much appreciate it.